demo-simple/src/main/java/com/novoda/downloadmanager/demo/MainActivity.java

```diff
@@ -35,6 +36,10 @@
 public class MainActivity extends AppCompatActivity {
 
     private static final String TAG = MainActivity.class.getSimpleName();
+
+    @SuppressLint("SdCardPath")
+    private static final String V1_BASE_PATH = "/data/data/com.novoda.downloadmanager.demo.simple/files/Pictures/";
```

Comment:
Good question but this actually has nothing to do with the context, lint flags this because of the use of `data`. This `V1_BASE_PATH` is the base path that is used in the v1 database, we would only need to change this if we change the v1 database.

Reply:
Ahh right, I put the comment before reading the rest of the PR... it's OK to have it like this 👍